<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>613</width>
    <height>409</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Image Editing Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="openBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>import.png</normaloff>import.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>save.png</normaloff>save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cropBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>crop.png</normaloff>crop.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="enhancementBtn">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ENHANCEMENT</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/addborder.png</normaloff>:/addborder.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>FLIP</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="horizontalWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="verticalFlipBtn">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>vertical.png</normaloff>vertical.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="horizontalFlipBtn">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>horizontal.png</normaloff>horizontal.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ROTATE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="horizontalWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>leftRotate.png</normaloff>leftRotate.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>r.png</normaloff>r.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>BRIGHTNESS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="horizontalWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>plus.webp</normaloff>plus.webp</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>minus.ico</normaloff>minus.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CONTRAST</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="horizontalWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>plus.webp</normaloff>plus.webp</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>minus.ico</normaloff>minus.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="quitBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>exit.png</normaloff>exit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>744</width>
         <height>683</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="effect1">
         <property name="text">
          <string>Effect1</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/prev.png</normaloff>:/prev.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect2">
         <property name="text">
          <string>Effect2</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/next.png</normaloff>:/next.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect3">
         <property name="text">
          <string>Effect3</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/zoomin.png</normaloff>:/zoomin.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect4">
         <property name="text">
          <string>Effect4</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/zoomout.png</normaloff>:/zoomout.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect5">
         <property name="text">
          <string>Effect5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect6">
         <property name="text">
          <string>Effect6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect7">
         <property name="text">
          <string>Effect7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect8">
         <property name="text">
          <string>Effect8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect9">
         <property name="text">
          <string>Effect9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect10">
         <property name="text">
          <string>Effect10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect11">
         <property name="text">
          <string>Effect11</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect12">
         <property name="text">
          <string>Effect12</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect13">
         <property name="text">
          <string>Effect13</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect14">
         <property name="text">
          <string>Effect14</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect15">
         <property name="text">
          <string>Effect15</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/originalsize.png</normaloff>:/originalsize.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect16">
         <property name="text">
          <string>Effect16</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect17">
         <property name="text">
          <string>Effect17</string>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect18">
         <property name="text">
          <string>Effect18</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/rotateleft.png</normaloff>:/rotateleft.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="effect19">
         <property name="text">
          <string>Effect19</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/rotateright.png</normaloff>:/rotateright.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="slideShowBtn">
         <property name="text">
          <string>Effect20</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/play.png</normaloff>:/play.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
